### ISSUE TITLE
Need to bump alpine version again.

### ISSUE THREAD

**Issue by jayped007** [tsub/docker-graph-easy#15]
docker build is currently failing, I suspect due to the specified alpine version

**Comment by tsub** [comment]
Indeed, this repository has not been maintained for a long time and the alpine version is very old, but the docker build was successful in my local machine.

@jayped007 Can I ask you to post a log of what error you got with the docker build?
